.github/workflows/lint.yaml

```diff
@@ -21,3 +21,12 @@ jobs:
 
       - run: pnpm lint
       - run: pnpm prettier --check .
+  clang-format-check:
+    name: clang-format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: sudo apt-get install clang-format
+      - run: |
+          chmod +x ./.github/scripts/clang-format-check-all.sh
```

Comment:
Added the chmod +x just in case the script somehow loses it's execute permission (has happened to me before - deleting, moving, copying files, possibly on different operating systems)